makefiles/Makefile.docs

```diff
@@ -53,7 +53,7 @@ next-gen-parse: examples
 		fi \
 	fi
 
-next-gen-html:  get-build-dependencies next-gen-parse 
+next-gen-html:  get-build-dependencies
```

Comment:
Is `get-build-dependencies` something we need to split out rn as well? Or is it okay to keep as-is based on your changes to the Autobuilder?

Reply:
@rayangler actually we could split this out rn as well. Because with the auto builder changes it will end up calling it twice because it is caused in the auto builder. But that has me thinking, was that always the case?

Reply:
Might've always been the case then 🤔

Reply:
It got me thinking as well.